2/src/part2.c

```diff
@@ -0,0 +1,74 @@
+// Link: https://adventofcode.com/2022/day/2
+
+#include <stdio.h>
+
+enum {
+    ROCK = 1,
+    PAPER,
+    SCISSORS
+};
+
+int decode_move(char move) {
+    switch (move) {
+        case 'A':  // Rock
+            return ROCK;
+        case 'B':  // Paper
+            return PAPER;
+        case 'C':  // Scissors
+            return SCISSORS;
+        default:
+            return 0;
+    }
+}
+
+int choose_your_move(int opponent_move, char desired_outcome) {
```

Comment:
why do we need this? very complicated logic.
to know your move you can use **decode_move**  function